testfiles/5ZoneEndUses.idf

```diff
@@ -856,7 +856,8 @@
     1,                       !- Type
     1,                       !- Multiplier
     2.438400269,             !- Ceiling Height {m}
-    239.247360229;           !- Volume {m3}
+    239.247360229,           !- Volume {m3}
+    90.0;                    !- Floor Area {m2}
```

Comment:
Add a couple of zone floor area overrides to a test suite file (since there weren't any before).

Reply:
And a warning now shows up in 5ZoneEndUses since calculated floor area is 98.1.